.prettierrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tabWidth": 2,
+  "printWidth": 80,
+  "arrowParens": "avoid",
```

Comment:
Nitpick:
I think the existing code uses a lot of arrowParens.
There's a lot of debate, between which is better. (I prefer avoid, but I've seen others always use it)

Do confirm with @IEEE-RVCE/frontend which they prefer

Reply:
Okay 👌🏼

Reply:
looks good